pages/common/cmake.md

```diff
@@ -1,13 +1,21 @@
 # cmake
 
-> Cross-platform build system generator.
-> It generates Makefiles, Visual Studio projects or others, depending on the target system.
+> Cross-platform build, testing and packaging automation system.
+> CMake uses own shell-like syntax and generates recipes for several build systems.
 > More information: <https://cmake.org/cmake/help/latest/manual/cmake.1.html>.
 
-- Generate a Makefile and use it to compile a project in the same directory as the source:
+- Generate a build recipe in the current directory with CMakeLists.txt from a project directory:
 
-`cmake && make`
+`cmake {{path/to/project_dir/}}`
```

Comment:
It's better not to have a trailing slash and to use the full name for "directory."
```suggestion
`cmake {{path/to/project_directory}}`
```

Reply:
@bl-ue, why? The trailing slash points that path is actually a directory, not a file. I believe it is good practice to end directory paths with slash in scripts, it is explicit and prevents errors on concatenation.

Reply:
@bergentroll I actually completely agree with you, however, it's long been the convention in the project to omit the trailing slash, so I think it would be best to follow convention.

For example, see [`pages/common/7z.md`](https://github.com/tldr-pages/tldr/blob/master/pages/common/7z.md).

Reply:
@bl-ue, well, with respect, I'd propose to revise this convention, because change will brake nothing. But I follow your decision.

Reply:
@bergentroll I'd recommend asking @sbrl and @waldyrious about that on [Gitter](https://gitter.im/tldr-pages/tldr).